policy/overview-policy-build-time-dependency-vulns.md

```diff
@@ -0,0 +1,120 @@
+including the management of fallout from any vulnerabilities found within those
+dependencies. This problem is only made worse by large dependency trees that are
+common in many software ecosystems. In this regard, there are a number of issues
+to tackle, with the space of solutions for them being intertwined:
+
+1. As a maintainer of a library, how do you ensure that vulnerabilities in your
+   library (found by third-parties) are reported to you securely?
+2. As a developer of a library, how do you advertise the discovery of a
+   vulnerability to your dependents?
+3. As a consumer of other libraries, how can you tell if one of your
+   dependencies is affected by a vulnerability?
+4. As a consumer of other libraries, how do you mitigate the effect of a
+   vulnerability in a dependency? Does this change with how far up the
+   dependency tree the issue is? How do you advertise the discovery of such
+   vulnerabilities?
+
+This document is intended as a high-level overview of policies for package
+maintainers who have responsibilities for libraries and/or dependent projects.
+For another take on the subject, see [this
+article](https://cheatsheetseries.owasp.org/cheatsheets/Vulnerable_Dependency_Management_Cheat_Sheet.html).
+
+## Mitigations
+
+In the long run, the hope would be that the solutions to all of these issues
+will be standardized across all software ecosystems. Users and developers alike
+should find it facile to identify the issues in question. However, only some of
+these issues have a suitable answer currently, and in many cases the answer is
+ecosystem-dependent.
+
+### Having vulnerabilities in your library reported to you
+
+Vulnerabilities in your system should be reportable in a secure way that
+minimizes risk to existing users while you implement fixes or mitigations.
+
+While some vulnerabilities will be discovered by the developers and maintainers
+of the library, others might be uncovered by researchers, users, or other third
+parties. A secure, private channel must exist for them to disclose their
+findings and seek advice or information on further steps. The channel must be
+closely monitored, and should be clearly advertised as part of the security
+policy.
+
+Reports should be thoroughly investigated, with frequent updates to the
+reporting party, and acknowledgement of their contribution following the public
+disclosure.
+
+More in-depth information can be found in [the guide for coordinated
+vulnerability disclosure for open source
+software](https://github.com/ossf/oss-vulnerability-guide) created by the
+OpenSSF.
+
+### Disclosing vulnerabilities in your library
+
+Disclosure of vulnerabilities should generally be done in a standardized fashion
+to allow downstream users to identify issues automatically regardless of how far
+the library is on the dependency chain. The [OpenSSF
+guide](policy/overview-policy-build-time-dependency-vulns.md) mentioned
```

Comment:
Not sure if this hyperlink is needed. It is also failing spellcheck and lint, fwiw.